unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -99,9 +122,33 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
                     entity.gameObject.transform.localRotation, true);
             }
 
-            avatarRenderer.ApplyModel(model, () => avatarDone = true, () => avatarFailed = true);
+            var wearableItems = model.wearables.ToList();
+            wearableItems.Add(model.bodyShape);
+            if (avatar.status != IAvatar.Status.Loaded || needsLoading)
+            {
+                //TODO Add Collider to the AvatarSystem
+                //TODO Without this the collider could get triggered disabling the avatar container,
```

Comment:
Should these be tackled before merging?

Reply:
This would extend the refactor a bit longer, I rather tackle it down later